### PR TITLE
Added integration docs

### LINKED ISSUE #64: Als functioneel beheerder wil ik weten hoe vernietiging van gerelateerde data buiten de ZRC en DRC zijn werk gaat,
zodat ik relevante user stories op kan stellen op dit vlak.
Ik denk in ieder geval aan notificaties over vernietiging naar aangesloten systemen, zodat ook in die systemen vernietigd kan worden. Met welke notificaties moet een leverancier van een ander systeem uit de voeten kunnen?  

[thema uit concept offerte: 8. Vernietigen van gerelateerde data in andere systemen documenteren en inhaak-systeem opzetten.]

*Clearification*

Het moet duidelijk worden hoe andere applicaties/componenten moeten omgaan met gegevens bij een zaak, als deze zaak wordt vernietigd. Samengevat:

1. VL uitvoeren
2. Zaak wordt vernietigd
3. Andere applicaties ontvangen een notificatie dat de zaak is vernietigd
4. Andere applicaties moeten dan hun gegevens ook vernietigen

Dit documenteren.

### PR DESCRIPTION
**Pull request by joeribekker**
Fixes #64

### CODE REVIEW

File: doc/manual/faq.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+What is exactly is deleted by the application?
```
Comment (SilviaAmAm) [inline doc/manual/faq.rst:6]:
```suggestion
What exactly is deleted by the application?
```

File: doc/manual/faq.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Yes! Deleting or destroying cases (zaken) will remove the record from the 
+database. It's not just a flag that is set on the deleted case.
+
+I accidentally deleted a case, can I get it back?
+-------------------------------------------------
+
+No! Unless you have a backup of the data, you cannot undo a deletion or get the
+data back.
+
+Is related data, not part of the API's voor Zaakgericht Werken, deleted?
```
Comment (SilviaAmAm) [inline doc/manual/faq.rst:36]:
```suggestion
Is related data, not part of the `API's voor Zaakgericht Werken`_, deleted?
```

File: doc/manual/faq.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+------------------------------------------------------------------------
+
+The application destroys cases and related documents and decisions in the 
+`API's voor Zaakgericht Werken`_. The application has no direct control over
+data that is stored in other applications, registers, API's or components.
```
Comment (SilviaAmAm) [inline doc/manual/faq.rst:41]:
```suggestion
data that is stored in other applications, registers, APIs or components.
```

File: doc/manual/faq.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+It's the responsibility of the recordmanager or archivist to make sure that
```
Comment (SilviaAmAm) [inline doc/manual/faq.rst:43]:
```suggestion
It's the responsibility of the record manager or archivist to make sure that 
```

File: doc/manual/step-by-step.rst [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
+============================
+Step by step process (Dutch)
+============================
 
 .. note:: This section is in Dutch and follows the same steps as the :ref:`demo`.
 
+Hieronder staat een typische workflow door het vernietigingsproces zoals dat
+mogelijk wordt gemaakt met deze applicatie. Het is mogelijk om het proces iets
+anders in te richten met behoudt van dezelfde functionaliteiten. Deze globale
+functionaliteiten staat beschreven in :ref:`destruction`.
```
Comment (SilviaAmAm) [inline doc/manual/step-by-step.rst:10]: Is it intended to have "functionaliteiten" plural and "staat" singular?

Comment (joeribekker) [reply doc/manual/step-by-step.rst:10]: *snif* Your Dutch is better than mine

**Review by SilviaAmAm — APPROVED** [review]

**Pull request merged**